tests/foreman/cli/test_hostgroup.py

```diff
@@ -311,6 +311,32 @@ def test_negative_delete_by_id():
         HostGroup.delete({'id': entity_id})
 
 
+@pytest.mark.tier2
+def test_positive_created_nested_hostgroup(module_org):
+    """Create a nested host group using multiple parent hostgroup paths.
+    e.g. ` hostgroup create --organization 'org_name' --name new3 --parent-title new_1/new_2`
+
+    :id: 02123cec-cb64-43dd-aafc-19332e2f5de7
+
+    :BZ: 1807536
+
+    :customerscenario: true
+
+    :CaseImportance: low
+
+    """
+    parent_hg = make_hostgroup(
+        {
+            'organization-ids': module_org.id,
+        }
+    )
```

Comment:
Optional:
```suggestion
    parent_hg = make_hostgroup({'organization-ids': module_org.id})
```

Reply:
Yes, Just wanted to make sure that it works with nested HG as well